src/renderer/services/clients/balances.ts

```diff
@@ -44,6 +44,7 @@ const loadBalances$ = ({
       (walletAddress) =>
         Rx.from(client.getBalance(walletAddress)).pipe(
           map(RD.success),
+          liveData.map(A.filter((balance) => !balance.amount.amount().isZero())),
```

Comment:
Please note: We don't want to exclude zero balances everywhere in the ui. For example: If an user has sent all amount of an token using `SEND` view, we might want to show balances there (which are 0 now). 

If we do need a filter for zero balances in asset view, do such filter there (but not here in this service here  ^).